client/src/components/SpecialBirdButton.js

```diff
@@ -0,0 +1,25 @@
+// import styled from '@emotion/styled';
+// import Klecks from '../ressources/Klecks.png';
+// import React from 'react';
+// import BirdIcon from '../icons/BirdIcon';
+
+// const SpecialButton = styled.button`
+//   display: flex;
+//   height: 80px;
+//   width: 80px;
+//   justify-content: center;
+//   align-items: center;
+//   background: ${Klecks};
+//   border: none;
+//   outline: none;
+// `;
+
+// function SpecialBirdButton() {
+//   return (
+//     <SpecialButton>
+//       <BigBirdIcon />
+//     </SpecialButton>
+//   );
+// }
+
+// export default SpecialBirdButton;
```

Comment:
Outcommented, do you actually need it? :)
NEver leave outcommented code unattended or it will be on github propably :D

Reply:
I will come back and work on a special button later. So I will keep it for now. Thank you.